### ISSUE TITLE
Parsing returns String objects

### ISSUE THREAD

**Issue by atd** [mwunsch/prism#3]
```
    Prism.find 'http://verselogic.net/' #=> last 2 objects are strings
```

I guess this is a bug. Using prism-0.1.0 gem

**Comment by mwunsch** [comment]
Can reproduce this with prism-0.1.0, but not master. Try cloning master and see if you get the same results.

**Comment by atd** [comment]
Same behavior, but different order:
        Prism.find('http://verselogic.net/').map(&:class).uniq #=> [Prism::Microformat::XFN, Prism::Microformat::HCard, String, Prism::Microformat::RelTag]

**Comment by mwunsch** [comment]
Isolated this to a problem with XOXO

**Comment by mwunsch** [comment]
Proper extraction of XOXO. Sort of kind of closed by 2cc92442498ec96af78b78b9499378f57f45b324
